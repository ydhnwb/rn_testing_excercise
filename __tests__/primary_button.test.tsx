import React from "react";
import renderer from 'react-test-renderer';
import PrimaryButton from "../src/presentation/shared_component/primary_button";

it('renders correctly', () => {
    const tree = renderer.create(<PrimaryButton/>);
    expect(tree).toMatchSnapshot()
})