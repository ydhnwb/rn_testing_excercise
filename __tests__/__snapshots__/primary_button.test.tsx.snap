// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "padding": 16,
      }
    }
  >
    <Text>
      Im a button
    </Text>
  </View>
</View>
`;
